File: 3226-minimum-number-game/minimum-number-game.swift
class Solution {
    func numberGame(_ nums: [Int]) -> [Int] {
        var arr: [Int] = []
        var sNums = nums.sorted()

        for i in stride(from: 0, to: sNums.count, by: 2) {

    arr.append(sNums[i + 1])
    arr.append(sNums[i])
}

        return arr
  }
}

File: 3226-minimum-number-game/minimum-number-game.py
class Solution:
    def numberGame(self, nums: List[int]) -> List[int]:

        sNums = sorted(nums)
        arr: List[int] = []


        for i in range(0, len(sNums), 2):
            arr.append(sNums[i+1])
            arr.append(sNums[i])
        
        return arr